README.md

```diff
@@ -1,5 +1,13 @@
 # BEA Adapter
 
+## Run in Docker
+
+- make docker and run
+```
+make docker
+docker run -p 8080:8080 -e API_KEY=<yourapikey> smartcontractkit/bea-adapter
```

Comment:
This might just be a personal opinion, but I think we should clearly differentiate between local and public tags. Suggestion being to change the tag to just `bea-adapter`.

Reply:
If it's in preparation of public release, then `make docker` should not be necessary (as well as org name is `smartcontract`).

Reply:
Whats your preference? For node ops it would be a lot easier if they can just use the docker from your org. If you won't be doing that then the first option makes more sense :) Pinging u on TG to discuss..